acceptance-tests/src/main/kotlin/org/ostelco/at/common/StripePayment.kt

```diff
@@ -121,16 +148,21 @@ object StripePayment {
     }
 
     fun deleteCustomer(customerId: String) {
+
         // https://stripe.com/docs/api/java#create_card_token
         Stripe.apiKey = System.getenv("STRIPE_API_KEY")
```

Comment:
Getting the stripe API key should be wrapped in a method that checks if it has a valid value (at least that it isn't empty), and throws/signals the appropriate error if it's not valid.   The rationale behind this request is: This will fail eventually, so it is prudent to fail as early as possible to make sure that half-committed values are not stored anywhere before the error is detected.